src/master-playlist-controller.js

```diff
@@ -371,6 +390,14 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.blacklistCurrentPlaylist(this.mainSegmentLoader_.error());
     });
 
+    this.mainSegmentLoader_.on('syncinfoupdate', () => {
+      this.onSyncInfoUpdate_();
+    });
+
+    this.audioSegmentLoader_.on('syncinfoupdate', () => {
+      this.onSyncInfoUpdate_();
+    });
```

Comment:
The current implementation of seekable uses both the main playlist and audio playlist, taking the intersection of the two as the seekable range, so I wanted to keep behavior as clsoe as possible. If the main playlist had a seekable window of 50 - 150 seconds, but the audio playlist had a seekable window of 75 - 125 seconds, I think it would make sense to not allow seeking outside of the audio range.